### DISCUSSION TITLE
Backup issue

### DISCUSSION THREAD

**Discussion by davidovic999** [mikeberger/borg_calendar#203] [General]
Hi Mike,
I had already created the backup folder and specified the path in the settings.
To backup, I've tried both the "Prompt for backup action" and "Backup to a file" options, but neither works.
There are no backup files in the backup folder! It's empty!

![backup-1](https://github.com/mikeberger/borg_calendar/assets/59475468/63cedd30-75ea-4132-8555-4aedf1cd8c53)

![backup](https://github.com/mikeberger/borg_calendar/assets/59475468/6f84d41a-0420-4b05-8fa9-eea7c140c9df)

---

I got this message on one of the start-ups:

![borg](https://github.com/mikeberger/borg_calendar/assets/59475468/95eabc3e-0e12-4dd0-be6e-6bc09393fc35)

I had clicked on "OK" and confirmed the path to the database in the window that followed.

**Comment by mikeberger** [comment]
BORG probably didn't have permissions to your backup folder. Change the backup folder to somewhere that doesn't require ADMIN privileges. BORG will never ask for ADMIN privileges. Since a shutdown was in progress, the program just probably wrote an error to .borg.out or .borg.log in your user folder saying that access was denied to the backup file. The files under C:\Program Files are meant to be read-only.

The database in use error is usually a race condition when starting 2 BORGs very quickly. Normally, when you try to start a second copy of BORG, it detects that a copy is running and just opens the copy that is already running.

**Comment by davidovic999** [comment]
Thank you for your reply.
The backup now works. I have created a backup folder in another partition.

Greetings,
